repo/packages/S/spark/13/config.json

```diff
@@ -0,0 +1,235 @@
+{
+    "type": "object",
+    "properties": {
+        "service": {
+            "description": "DCOS Spark configuration properties",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "default": "spark",
+                    "description": "The Spark Dispatcher will register with Mesos with this as a framework name.  This service will be available at http://<dcos_url>/service/<name>/",
+                    "type": "string"
+                },
+                "cpus": {
+                    "default": 1,
+                    "description": "CPU shares",
+                    "minimum": 0.0,
+                    "type": "number"
+                },
+                "mem": {
+                    "default": 1024.0,
+                    "description": "Memory (MB)",
+                    "minimum": 1024.0,
+                    "type": "number"
+                },
+                "role": {
+                    "description": "The Spark Dispatcher will register with Mesos with this role.",
+                    "type": "string",
+                    "default": "*"
+                },
+                "principal": {
+                    "description": "The Spark Dispatcher will register with Mesos with this principal.",
+                    "type": "string",
+                    "default": ""
+                },
+                "secret": {
+                    "description": "The Spark Dispatcher will register with mesos with this secret.",
+                    "type": "string",
+                    "default": ""
+                },
```

Comment:
This principal and secret are passed to the Credentials object that's passed to the MesosSchedulerDriver.  I don't know if this is CRAM_MD5.  Let me know how to use this for EE authentication if you'd like to try that.